recipes-core/aziot-edged/aziot-edged.inc

```diff
@@ -1,8 +1,9 @@
-DEPENDS += "openssl virtual/docker iotedge aziotd systemd"
-RDEPENDS_${PN} += "docker iotedge aziotd aziot-keys systemd"
+DEPENDS += "openssl virtual/docker iotedge aziotd"
+RDEPENDS_${PN} += "docker iotedge aziotd aziot-keys"
 
 inherit systemd
 
+SYSTEMD_PACKAGES = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', '${PN}', '', d)}"
```

Comment:
@maskimthedog This should help find systemd in the distro right? this is the method we followed in 1.1

Reply:
That if systemd is enabled in the DISTRO_FEATURES.

Reply:
> @maskimthedog This should help find systemd in the distro right? this is the method we followed in 1.1

That line just sets a variable based on the presence or lack thereof of systemd in DISTRO_FEATURES.

Reply:
> > @maskimthedog This should help find systemd in the distro right? this is the method we followed in 1.1
> 
> That line just sets a variable based on the presence or lack thereof of systemd in DISTRO_FEATURES.

You need it if the recipe needs to decide whether to install sysvinit or systemd service based on the DISTRO_FEATURES setting of systemd.

Reply:
> > > @maskimthedog This should help find systemd in the distro right? this is the method we followed in 1.1
> > 
> > 
> > That line just sets a variable based on the presence or lack thereof of systemd in DISTRO_FEATURES.
> 
> You need it if the recipe needs to decide whether to install sysvinit or systemd service based on the DISTRO_FEATURES setting of systemd.

We only care about systemd.

Reply:
https://git.yoctoproject.org/poky/plain/meta/recipes-core/systemd/ then need to update the receipe to add this?

Reply:
> https://git.yoctoproject.org/poky/plain/meta/recipes-core/systemd/ then need to update the receipe to add this?

@ggjjj Just need this...

inherit systemd

SYSTEMD_AUTO_ENABLE_${PN} = "enable"
SYSTEMD_SERVICE_${PN} = "aziot-edged.service"